### PR TITLE
[WIP] workflow enhancements

### PR DESCRIPTION
**Pull request by zachwe**

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -3,10 +3,16 @@ FROM gcr.io/deeplearning-platform-release/pytorch-gpu.1-10
 WORKDIR /root
 
 RUN apt-get update && apt-get install -y espeak libsndfile-dev
-RUN pip install git+https://github.com/uberduck-ai/uberduck-ml-dev
+
+RUN pip install git+https://github.com/uberduck-ai/uberduck-ml-dev@zach-workflow-enhancements
```
Comment (sjkoelle) [inline docker/Dockerfile:7]: fix before merge?

File: docker/entrypoint.sh [hunk 2]
```diff
@@ -1,28 +1,85 @@
...
+    export BUCKET=$1
+fi
+if [[ -z "$2" ]]; then
+    echo "Enter Experiemnts repo to clone: "
+    read inp
+    export UBMLEXP_GIT=$inp
+else
+    export UBMLEXP_GIT=$2
+fi
+if [[ -z "$3" ]]; then
```
Comment (sjkoelle) [inline docker/entrypoint.sh:18]: does this mean than argument 3 always has to be warm start name?  what if I want arguments 2 and 4?

**Review by sjkoelle — APPROVED** [review]

File: docker/entrypoint.sh (hunk 2 above)
Comment (zachwe) [reply docker/entrypoint.sh:18]: Unfortunately not supported in this branch .... I think we could add command line flags to get that

File: docker/Dockerfile (hunk 1 above)
Comment (zachwe) [reply docker/Dockerfile:7]: I have some additional changes that are in flight, so I will just keep this and continue developing in this branch

**Pull request merged**